generation/new_client_hermetic_build/README.md

```diff
@@ -28,6 +28,13 @@ in the Buganizer request.
 The example in this README uses AlloyDB's [Cloud Drop](https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/alloydb_v1.yaml)
 file as an example.
 
+
+:warning: **IMPORTANT:**
+Not all the `new-client.py` arguments are available in the Github Action.
+Please refer to
+[this
+section](https://github.com/googleapis/google-cloud-java/blob/main/generation/new_client_hermetic_build/README.md#advanced-options)
```

Comment:
Could you use local reference, starting with `/`?

Reply:
Done